VideoLocker/res/layout/row_course_list.xml

```diff
@@ -156,7 +159,8 @@
             android:layout_alignParentRight="true"
             android:layout_alignTop="@id/schoolcode_layout"
             android:layout_below="@id/course_name"
-            android:layout_marginRight="10dp" >
+            android:layout_marginRight="10dp"
+            android:layout_toRightOf="@id/schoolcode_layout">
```

Comment:
The right most view does have variation, as it is dependent on the month names. 
I will reorder the code to use <code>toLeftOf</code>